pkgs/applications/editors/neovim/default.nix

```diff
@@ -108,6 +108,7 @@ let
                 $out/bin/nvim
     '' + optionalString withPython ''
       ln -s ${pythonEnv}/bin/python $out/bin/nvim-python
+      ln -s ${pythonEnv}/bin/python $out/bin/python2
```

Comment:
When I disable python3 for neovim this PR works for me, including the plugin :-)